docs/cvl/methods.md

```diff
@@ -57,36 +65,131 @@ method_summary   ::= "ALWAYS" "(" value ")"
                    | "DISPATCHER" [ "(" ( "true" | "false" ) ")" ]
                    | "AUTO"
                    | id "(" [ id { "," id } ] ")"
+```
+
+See {doc}`types` for the `evm_type` production.  See {doc}`basics`
+for the `id` production.  See {doc}`expr` for the `expression` production.
+
+Methods entry patterns
+----------------------
+
+Each entry in the methods block contains a pattern that matches some set of
+contract functions.
+
+ - {ref}`exact-methods-entries` match a single method of a single contract.
+ - {ref}`wildcard-methods-entries` match a single method signature on all contracts.
```

Comment:
There's also catch-all now

Reply:
yes, I had initially added it, but realized I didn't know the details and that it would make sense as a separate PR

Reply:
I have it mentioned in a very cursory fashion in the managing timeouts section, but definitely with room for expansion